string table_name
---
bool success
